kitsune/products/tests/test_templates.py

```diff
@@ -158,3 +159,46 @@ def test_subtopics(self):
         eq_(200, r.status_code)
         pqdoc = pq(r.content)
         eq_(1, len(pqdoc('li.subtopic')))
+
+    def test_community_support_shown_for_default_language(self):
+        """Verify the get community support is shown in the topics for en-US."""
+        p = ProductFactory()
+
```

Comment:
So I think, as there are no topic is created for the product, the ``topics`` become ``None``. Thats why the test is failing.
Lets create a Topic by ``TopicFactory(product=p, visible=True)``